tests/test_transform/test_transform_scheme.py

```diff
@@ -72,28 +72,3 @@ def test_multiple_groups():
     assert not scheme.randomize
     assert scheme.type == "hadamard"
     assert len(scheme.apply) == 20
-
-
-def test_transform_scheme_block_size():
-    """
-    Ensure json with (deprecated) `head_dim` or `block_size`
-    both load up correctly and save with `block_size` field
-    """
-
-    old_scheme = TransformScheme.model_validate_json(
-        '{"type": "hadamard", "head_dim": 128}'
-    )
```

Comment:
Shouldn't we still be testing that this gets loaded correctly? Or do we have tests for this elsewhere?